crates/goose-server/src/routes/mcp_ui_proxy.rs

```diff
@@ -0,0 +1,167 @@
+use axum::{
+    http::header,
+    response::{Html, IntoResponse, Response},
+    routing::get,
+    Router,
+};
+
+const MCP_UI_PROXY_HTML: &str = r#"<!doctype html>
```

Comment:
To make this easier to maintain, can we keep this HTML as a standalone file and inline it like we do in the [autovisualiser mod.rs](https://github.com/block/goose/blob/feat/mcp-ui-improvements-rust-server-goosed/crates/goose-mcp/src/autovisualiser/mod.rs#L1086)?